testdata/default/cheats/ExpectEmit.t.sol

```diff
@@ -578,6 +584,70 @@ contract ExpectEmitTest is DSTest {
         emitter.emitWindow();
     }
 
+    /// ----- `count` arg tests -----
+
+    /// Test zero emits.
+    function testCountNoEmit() public {
+        vm.expectEmit(0);
+        emit Something(1, 2, 3, 4);
+        emitter.doesNothing();
+    }
+
+    function testFailNoEmit() public {
+        vm.expectEmit(0);
+        emit Something(1, 2, 3, 4);
+        emitter.emitEvent(1, 2, 3, 4);
+    }
```

Comment:
With the upcoming goal of deprecating `testFail` I think it would be worth exploring if we can avoid it

Is this something we can capture with `expectRevert` to make more strict assertions?

Reply:
Not a requirement for getting this merged, more of a general question